react/src/typings.tsx

```diff
@@ -250,3 +250,29 @@ export interface Option {
     disabled?: boolean;
     selected?: boolean;
 }
+
+export interface Gene {
+    gene_id: number;
+    hgnc_gene_id: number;
+    ensembl_id: number;
+    gene: string;
+    hgnc_gene_name: string;
+    variants?: Variant[];
```

Comment:
These are all nullable but we might be changing this a bit given the discussion today.

Reply:
updated in [bec4861](https://github.com/ccmbioinfo/stager/commit/bec48617015303e9dc4870e8a56d498c8052d646)